cmd/katapult/config_test.go

```diff
@@ -41,6 +36,12 @@ func TestConfig(t *testing.T) {
 			apiKey: "",
 			apiURL: "test",
 		},
+		{
+			name:   "test JSON output",
+			apiKey: "test",
+			apiURL: "test",
+			output: "json",
+		},
 	}
 	for _, tt := range tests {
 		t.Run(fmt.Sprintf("key: %s, url: %s", tt.apiKey, tt.apiURL), func(t *testing.T) {
```

Comment:
We should probably change the sub-test to use `tt.name` to convey more information in test output and golden filenames:

```suggestion
		t.Run(tt.name, func(t *testing.T) {
```